### DISCUSSION TITLE
Connection EGM with C++ on Windows

### DISCUSSION THREAD

**Discussion by YKK1024** [ros-industrial/abb_libegm#140] [General]
Hello until now we have successfully apply this library with ROS.
But I still want to know if there is an interface between the EGM with Windows (based maybe on C++)

Thank you in advance.

**Comment by gavanderhoorn** [comment]
`abb_libegm` is a plain CMake project. It does not use anything of ROS, nor does it offer ROS anything directly.

You should be able to use it from any other C/C++ project.
